### PR TITLE
docs(hive): add docs about how to deploy and query hive sqls

### PR DESCRIPTION
**Pull request by sandflee**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

### CODE REVIEW

**Comment by sandflee** [comment]
cc @wubx

**Comment by sandflee** [comment]
databend-with-hive release url  could be refreshed after https://github.com/datafuselabs/databend/issues/8649

File: website/blog/2022-11-28-hive.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+```
+[storage]
+type = "hdfs"
+
+[storage.hdfs]
+# hdfs namenode address,such as 127.0.0.1:8020
+name_node = "xx"
+root = ""
+
+[catalog]
```
Comment (Xuanwo) [inline website/blog/2022-11-28-hive.md:37]: `catalog` has been deprecated. Maybe we can migrate to `catalogs`.

```toml
[catalogs.hive]
type = "hive"
address = "xx"
```

**Comment by PsiACE** [comment]
@mergify update

**Comment by sandflee** [comment]
@mergify update

**Comment by Xuanwo** [comment]
Thanks for contribution!

**Pull request merged**